### PR TITLE
Monitoring: Regeneration

### PR DESCRIPTION
**Pull request by landrito**

### CODE REVIEW

File: packages/monitoring/package.json [hunk 1]
```diff
@@ -4,10 +4,37 @@
   "version": "0.3.0",
   "author": "Google Inc",
   "description": "Stackdriver Monitoring API client for Node.js",
+  "contributors": [
```
Comment (swcloud) [inline packages/monitoring/package.json]: We don't need these hand-edits at all. The library is auto-generated. So there will be no 'contributors' and 'CONTRIBUTING'.

Please keep this file as is.

File: packages/monitoring/src/v3/index.js [hunk 2]
```diff
@@ -39,4 +39,4 @@ v3.ALL_SCOPES = union(
   metricServiceClient.ALL_SCOPES
 );
 
-module.exports = v3;
\ No newline at end of file
+module.exports = v3;
```
Comment (swcloud) [inline packages/monitoring/src/v3/index.js]: This needs to be fixed in toolkit.

File: packages/monitoring/smoke-test/metric_service_smoke_test.js [hunk 3]
```diff
@@ -89,4 +89,4 @@ describe('MetricServiceSmokeTest', function() {
     .on('error', done)
     .on('end', done);
   });
-});
\ No newline at end of file
+});
```
Comment (swcloud) [inline packages/monitoring/smoke-test/metric_service_smoke_test.js]: This needs to be fixed in toolkit.

File: packages/monitoring/src/index.js [hunk 4]
```diff
@@ -102,4 +102,4 @@ module.exports.types = v3Protos;
 module.exports.v3 = {};
 module.exports.v3.group = groupV3;
 module.exports.v3.metric = metricV3;
-module.exports.v3.types = v3Protos;
\ No newline at end of file
+module.exports.v3.types = v3Protos;
```
Comment (swcloud) [inline packages/monitoring/src/index.js]: This needs to be fixed in toolkit.

File: packages/monitoring/smoke-test/metric_service_smoke_test.js [hunk 5]
```diff
@@ -17,7 +17,7 @@
 
 describe('MetricServiceSmokeTest', function() {
   if (!process.env.SMOKE_TEST_PROJECT) {
-    throw new Error("Usage: SMOKE_TEST_PROJECT=<project_id> node #{$0}");
+    throw new Error("Usage: SMOKE_TEST_PROJECT=<project_id> npm run smoke-test");
```
Comment (swcloud) [inline packages/monitoring/smoke-test/metric_service_smoke_test.js]: This needs to be fixed in toolkit.

File: packages/monitoring/src/v3/group_service_client.js [hunk 6]
```diff
@@ -35,7 +35,7 @@ var SERVICE_ADDRESS = 'monitoring.googleapis.com';
 
 var DEFAULT_SERVICE_PORT = 443;
 
-var CODE_GEN_NAME_VERSION = 'gapic/0.7.1';
+var CODE_GEN_NAME_VERSION = 'gapic/0.3.0';
```
Comment (swcloud) [inline packages/monitoring/src/v3/group_service_client.js]: This needs to be '0.7.1'

File: packages/monitoring/src/v3/index.js [hunk 7]
```diff
@@ -31,7 +32,11 @@ function v3(options) {
   return result;
 }
 
+v3.GAPIC_VERSION = '0.3.0';
```
Comment (swcloud) [inline packages/monitoring/src/v3/index.js]: This needs to be '0.7.1'

File: packages/monitoring/src/v3/metric_service_client.js [hunk 8]
```diff
@@ -35,7 +35,7 @@ var SERVICE_ADDRESS = 'monitoring.googleapis.com';
 
 var DEFAULT_SERVICE_PORT = 443;
 
-var CODE_GEN_NAME_VERSION = 'gapic/0.7.1';
+var CODE_GEN_NAME_VERSION = 'gapic/0.3.0';
```
Comment (swcloud) [inline packages/monitoring/src/v3/metric_service_client.js]: This needs to be '0.7.1'

**Comment by landrito** [comment]
@swcloud Made this simply the generated packages including the docs. I'll address the hand-edits in toolkit.

**Comment by swcloud** [comment]
@landrito LGTM except the GAPIC version number. @stephenplusplus may also need to approve it before he merges to the master.

File: packages/monitoring/smoke-test/metric_service_smoke_test.js [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+    var options = {autoPaginate: false};
+    function callback(responses) {
+        // The actual resources in a response.
+        var resources = responses[0];
+        // The next request if the response shows there's more responses.
+        var nextRequest = responses[1];
+        // The actual response object, if necessary.
+        // var rawResponse = responses[2];
+        for (var i = 0; i < resources.length; ++i) {
+            console.log(resources[i])
```
Comment (stephenplusplus) [inline packages/monitoring/smoke-test/metric_service_smoke_test.js]: > google-cloud-node@ lint C:\Users\sawch\dev\google-cloud-node
> jshint scripts/ packages/ system-test/ test/ && jscs packages/ system-test/ test/

packages\monitoring\smoke-test\metric_service_smoke_test.js: line 64, col 38, Missing semicolon.

File: packages/monitoring/src/index.js [hunk 10]
```diff
@@ -1,54 +1,105 @@
...
  *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+'use strict';
 
-/*!
```
Comment (stephenplusplus) [inline packages/monitoring/src/index.js]: Can we keep this block, and add a `@name` property:

```js
/*!
 * @module monitoring
 * @name Monitoring
 */
```

Our docs need this to know the proper-cased name of this module.

Comment (stephenplusplus) [inline packages/monitoring/src/index.js]: (PR for the docs parser side of this: #2429)

File: packages/monitoring/package.json [hunk 11]
```diff
@@ -48,19 +21,21 @@
...
-    "google-proto-files": "^0.12.0"
+    "google-proto-files": "^0.12.0",
+    "google-gax": "^0.13.2",
+    "lodash.union": "^4.6.0",
+    "extend": "^3.0.0"
   },
   "devDependencies": {
-    "mocha": "^3.1.2"
+    "mocha": "^3.2.0",
+    "through2": "^2.0.3"
```
Comment (stephenplusplus) [inline packages/monitoring/package.json]: [npm-check](http://gitnpm.com/npm-check)

```
$ npm-check

through2   😕  NOTUSED?  Still using through2?
                        Depcheck did not find code similar to require('through2') or import from 'through2'.
                        Check your code before removing as depcheck isn't able to foresee all ways dependencies can be used.
                        Use --skip-unused to skip this check.
                        To remove this package: npm uninstall --save-dev through2
```

**Pull request closed without merging**